src/components/SubmitResponsePage.jsx

```diff
@@ -39,6 +55,16 @@ const SubmitResponse = (props) => {
 											www.baltimorecountymd.gov/followup
 										</a>.
 									</p>
+									<div className="cs-form-control">
+										<ButtonDisplay
```

Comment:
Same thing se button

```js
<SeButton
  text="Create New Report"
  onClick={returnHome}
/>
```

Reply:
this is new, and i believe will eventually replace ButtonDisplay (i think)...i didn't realize that existed until this PR.